plugins/com.google.cloud.tools.eclipse.appengine.ui/plugin.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.4"?>
+<plugin>
+
+  <extension point="org.eclipse.ui.keywords">
+    <keyword label="GCP" id="com.google.cloud.tools.eclipse.appengine.newproject.GCPKeyword"/>
```

Comment:
- These labels should be put into the `plugin.properties` as they're internationalizable / searchable.
- The ids shouldn't have `new project`